deploy/charts/openebs-monitoring/dashboards/cStor/cstor-pool.json

```diff
@@ -0,0 +1,736 @@
+		"fillGradient": 0,
+		"gridPos": {
+			"h": 8,
+			"w": 12,
+			"x": 12,
+			"y": 16
+		},
+		"hiddenSeries": false,
+		"id": 12,
+		"legend": {
+			"avg": false,
+			"current": false,
+			"max": false,
+			"min": false,
+			"show": true,
+			"total": false,
+			"values": false
+		},
+		"lines": true,
+		"linewidth": 1,
+		"nullPointMode": "null",
+		"options": {
+			"alertThreshold": true
+		},
+		"percentage": false,
+		"pluginVersion": "7.5.5",
+		"pointradius": 2,
+		"points": false,
+		"renderer": "flot",
+		"seriesOverrides": [],
+		"spaceLength": 10,
+		"stack": false,
+		"steppedLine": false,
+		"targets": [
+			{
+			"exemplar": true,
+			"expr": "sum(openebs_total_write_count{pod=\"$cstor_pool\"})",
+			"interval": "",
+			"legendFormat": "wrie_iops",
+			"refId": "A"
+			}
+		],
+		"thresholds": [],
+		"timeFrom": null,
+		"timeRegions": [],
+		"timeShift": null,
+		"title": "Write IOPS",
+		"tooltip": {
+			"shared": true,
+			"sort": 0,
+			"value_type": "individual"
+		},
+		"type": "graph",
+		"xaxis": {
+			"buckets": null,
+			"mode": "time",
+			"name": null,
+			"show": true,
+			"values": []
+		},
+		"yaxes": [
+			{
+			"$$hashKey": "object:1625",
+			"format": "short",
+			"label": null,
+			"logBase": 1,
+			"max": null,
+			"min": null,
+			"show": true
+			},
+			{
+			"$$hashKey": "object:1626",
+			"format": "short",
+			"label": null,
+			"logBase": 1,
+			"max": null,
+			"min": null,
+			"show": false
+			}
+		],
+		"yaxis": {
+			"align": false,
+			"alignLevel": null
+		}
+		}
+	],
+	"refresh": "1m",
+	"schemaVersion": 27,
+	"style": "dark",
+	"tags": [
+		"OpenEBS"
+	],
+	"templating": {
+		"list": [
+		{
+			"current": {
+			"selected": true,
+			"text": "Prometheus",
+			"value": "Prometheus"
+			},
+			"description": null,
+			"error": null,
+			"hide": 0,
+			"includeAll": false,
+			"label": null,
+			"multi": false,
+			"name": "datasource",
+			"options": [],
+			"query": "prometheus",
+			"queryValue": "",
+			"refresh": 1,
+			"regex": "",
+			"skipUrlSync": false,
+			"type": "datasource"
+		},
+		{
+			"allValue": null,
+			"current": {},
+			"datasource": "$datasource",
+			"definition": "",
+			"description": null,
+			"error": null,
+			"hide": 0,
+			"includeAll": false,
+			"label": null,
+			"multi": false,
+			"name": "cstor_pool",
+			"options": [],
+			"query": "label_values(openebs_pool_status, pod)",
+			"refId": "StandardVariableQuery",
+			"refresh": 2,
+			"regex": "",
+			"skipUrlSync": false,
+			"sort": 0,
+			"tagValuesQuery": "",
+			"tags": [],
+			"tagsQuery": "",
+			"type": "query",
+			"useTags": false
+		}
+		]
+	},
+	"time": {
+		"from": "now-6h",
+		"to": "now"
+	},
+	"timepicker": {
+		"refresh_intervals": [
+		"5s",
+		"10s",
+		"30s",
+		"1m",
+		"5m",
+		"15m",
+		"30m",
+		"1h",
+		"2h",
+		"1d"
+		],
+		"time_options": [
+		"5m",
+		"15m",
+		"1h",
+		"6h",
+		"12h",
+		"24h",
+		"2d",
+		"7d",
+		"30d"
+		]
+	},
+	"timezone": "",
+	"title": "cStor Pool Dashboard",
```

Comment:
Can we also change the name to something like `cStor / Pool ` to categorise the dashboards for cstor?

Reply:
Changed title to:  `cStor / Pool / cStor Pool Dashboard`

Reply:
More better title would be `OpenEBS / cStor / Pool dashboard` in my opinion.